doctoshotgun.py

```diff
@@ -335,6 +335,17 @@ def find_centers(self, where, motives=None, page=1):
                 for center in self.find_centers(where, motives, next_page):
                     yield center
 
+        if additional_centers:
+            for additional_center in additional_centers:
+                splitted = additional_center.split(';')
+                if len(splitted) == 3:
+                    yield {
+                        "name_with_title": splitted[0], #"Corona Impfzentren - Berlin",
+                        "city": splitted[1], #"Berlin",
+                        "url": splitted[2], #"/institut/berlin/ciz-berlin-berlin"
+                    }
```

Comment:
I prefer the more explicit nature of named capture groups here, though both solutions do work just fine.
```suggestion
                match = re.match(r'(?P<name>.+);(?P<city>.+);(?P<url>.+)', additional_center)
                if match:
                    yield {
                        "name_with_title": match['name'], #"Corona Impfzentren - Berlin",
                        "city":            match['city'], #"Berlin",
                        "url":             match['url'],  #"/institut/berlin/ciz-berlin-berlin"
                    }
```